### PR TITLE
feat: OPX3.1-dev1

### PR DESCRIPTION
**Pull request by Tejaswi-Goel**
-   Added media configuration files
-   Added Dell EMC S5248F-ON
-   Added bcm_env.cfg with 32M dmasize for S5248-ON
-   Added ACL slice configuration to SAI initialization
-   Added start type determination
-   Refactored PSU fan speed mapping
-   Added support for external sensors in PAS
-   Updated SAI configuration for Dell EMC S4248FB(L)-ON
-   Updated SAI phy media and ACL profiles for Dell EMC S5148F-ON
-   Fix for Dell EMC S6010-ON LEDs
-   Fix for Dell EMC Z9264F-ON SAI configuration
-   Corrected SDI configuration for Dell EMC Z9264F-ON

Co-authored-by: Howard Persh <Howard_Persh@dell.com>
Signed-off-by: Tejaswi Goel <Tejaswi_Goel@Dell.com>

### CODE REVIEW

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now bf83735e2051a1c446347b222d7f9bb7ad66066b

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now fa64417807848b00aa8709330c434a074cb0135b

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 1624fb780fe237cd74ef74f8c8a01f89bbad1ecb

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,23 @@
+opx-platform-config (1.6.0) unstable; urgency=medium
+
+  * feat: miscellaneous updates
```
Comment (jeff-yin) [inline debian/changelog]: Please format CHANGELOG to look like the other pull-requests for OPX 3.1.0

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 5ee066167722f6bbd63b0a70eabe1d08b14e4720

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 6c846a2b2a9ab3eb72e610465d25f16085ec1a75

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 712cf65a2af05b998f28b83ff245f2491220b106

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+  * Update: Added Dell EMC S5248F-ON
+  * Update: Added bcm_env.cfg with 32M dmasize for S5248-ON
+  * Update: Updated copyrights
+  * Update: Added ACL slice configuration to SAI initialization
+  * Update: Added start type determination
+  * Update: Refactored PSU fan speed mapping
+  * Update: Added support for external sensors in PAS
+  * Update: Updated SAI configuration for Dell EMC S4248FB(L)-ON
+  * Update: Updated SAI phy media and ACL profiles for Dell EMC S5148F-ON
+  * Update: Fix for Dell EMC S6010-ON LEDs
```
Comment (jeff-yin) [inline debian/changelog]: These last 3 lines should be "Bugfix" entries.

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,21 @@
+opx-platform-config (1.6.0) unstable; urgency=medium
+
+  * Update: Added media configuration files
+  * Update: Added Dell EMC S5248F-ON
+  * Update: Added bcm_env.cfg with 32M dmasize for S5248-ON
+  * Update: Updated copyrights
```
Comment (jeff-yin) [inline debian/changelog]: You can remove stuff about updating copyrights.

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 736aebe2f7fde0261eae0d8f73ed626caf4a7e38

File: debian/changelog (hunk 3 above)
Comment (Tejaswi-Goel) [reply debian/changelog]: done

File: debian/changelog (hunk 2 above)
Comment (Tejaswi-Goel) [reply debian/changelog]: thank you, changed it.

**Pull request merged**